app/models/journeys/income/IncomeJourneyAnswers.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.journeys.income
+
+import models.{HowMuchTradingAllowance, TradingAllowance}
+import play.api.libs.json.{Json, OFormat}
+
+case class IncomeJourneyAnswers(incomeNotCountedAsTurnover: Boolean,
```

Comment:
it is fine, just one observation we may represent incorrect states:

incomeNotCountedAsTurnover=true
nonTurnoverIncomeAmount=None

ideally we should aim in not possible to represent incorrect states.

It's fine now, we can refine closer when we add validation

Reply:
That's a good point. I'll have a look at putting this constraint in this PR.

Reply:
We have agreed this will now be done in the backend